### PR TITLE
 Allow renaming of non fha trees

### PR DESCRIPTION
**Pull request by kostobog**
@blcham 
Fix partially #541

### CODE REVIEW

File: src/components/editor/faultTree/menu/faultTree/FaultTreeContextMenu.tsx [hunk 1]
```diff
@@ -35,7 +36,7 @@ const FaultTreeContextMenu = ({ anchorPosition, onClose, onEditClick, onDelete }
           : undefined
       }
     >
-      <MenuItem key="fault-tree-menu-rename" onClick={handleEditClick}>
+      <MenuItem disabled={!canRename} key="fault-tree-menu-rename" onClick={handleEditClick}>
```
Comment (blcham) [inline src/components/editor/faultTree/menu/faultTree/FaultTreeContextMenu.tsx:39]: Is it possible to put a tooltip that "FHA-based events cannot be renamed?"

Comment (blcham) [reply src/components/editor/faultTree/menu/faultTree/FaultTreeContextMenu.tsx:39]: @kostobog

Comment (kostobog) [reply src/components/editor/faultTree/menu/faultTree/FaultTreeContextMenu.tsx:39]: @blcham 
Yes, I think it is possible.

Comment (blcham) [reply src/components/editor/faultTree/menu/faultTree/FaultTreeContextMenu.tsx:39]: I created a new ticket for that -- https://github.com/kbss-cvut/fta-fmea-ui/issues/551

**Pull request merged**